app/server/ruby/lib/sonicpi/synths/synthinfo.rb

```diff
@@ -3290,6 +3290,214 @@ def arg_defaults
       end
     end
 
+    class WinwoodLead < SonicPiSynth
+      def name
+        "Winwood Lead"
+      end
+
+      def introduced
+        Version.new(3,3,0)
+      end
+
+      def synth_name
+        "winwood_lead"
+      end
+
+      def on_start(studio, args_h)
+        args_h[:rand_buf] = studio.rand_buf_id
+      end
+
+      def doc
+        "A lead synth inspired by the Winwood songs from the early 80s.  Adapted for Sonic Pi from [Steal This Sound](https://raw.githubusercontent.com/supercollider/supercollider/develop/examples/demonstrations/stealthissound.scd). Published there under [GPL v3](https://www.gnu.org/licenses/gpl-3.0.en.html), so re-published under the same terms. The source code is available [on the Sonic Pi GitHub repository](https://github.com/sonic-pi-net/sonic-pi). Date of modification: 10.01.2021"
```

Comment:
If I were to be _really_ picky, is an average user going to need to know directly from the GUI's synth docs exactly what license it is how or where to find the source code? We could leave notes like that in the sclang file maybe 🤷‍♂️  (perhaps it doesn't really matter if we leave it in, but I thought this was a reasonable question to ask 😅 🤷‍♂️)

Reply:
I was wondering about that, too, but I have tried to comply with [GPL v3](https://www.gnu.org/licenses/gpl-3.0.de.html), and that license explicitly requires that:

"5 d)  If the work has interactive user interfaces, each must display Appropriate Legal Notices; however, if the Program has interactive interfaces that do not display Appropriate Legal Notices, your work need not make them do so."

Reply:
@samaaron - shall we leave the license and source code link in the synth doc here given the above?

Reply:
We already have an explicit user interface component that shows that the app is distributed under a GPL license which is built into the GUI under a section called "License". I'm assuming that this is sufficient and we don't have to explicitly document it in the built-in documentation - although I do like having a link to the original designs so people can see the provenance.

Reply:
@level-xx would you be ok removing all the license stuff from the docs, keeping a URL to the original project and then adding an entry to the License.md file covering these synths?

Reply:
Sure, that's the best way. I was unaware of the license component. Just to avoid creating a mess ... I have noticed you have changed the base branch from `main` to `dev`. So, I would pull, merge my previous changes into `dev`, make the new license changes, and push to `dev`, correct?

Reply:
@level-xx @samaaron I suggest using GPL for the synthdefs and MIT for the synth metadata and documentation as Sam suggested in his recent comment on this PR. Regarding the target of this PR - you can just make a new PR with all the adjusted code ready to go, and targeted at `dev` 🙂